### PR TITLE
Revisions of documentation

### PR DESCRIPTION
**Pull request by heatherLstephens**
Herein I have revised the documentation to improve readability and clarity. Please do read through my edits, in particular in the Luna tips and tricks, as I'm not entirely certain I retained the original meaning properly in each revision made.

For the most part, I edited out usage of "you" or "we," but some remain. In general it is preferred to leave these out entirely in formal writing, but we can be a bit casual with this rule.

I also tried to capitalize Ansible and Docker where referred to in the body of text (as opposed to commands) but I probably missed some.

### CODE REVIEW

File: doc/compute_storage_schemes.rst [hunk 1]
```diff
@@ -36,15 +36,15 @@ The script only makes use of Linux utilities that are available at provisioning
 	mkdir /sysroot/boot
 	mount /dev/sda1 /sysroot/boot
 
-Sometimes, it might be desired to format the compute disks using a different filesystem that is not included in initial ramdisk in which the partitioning script is called. In such cases it is required to update the ``dracut`` configuration in the osimage to include the desired filesystem support and utilities then pack it up.
+It may be necessary to format the compute disks using a different filesystem not included in the initial ramdisk called by the partitioning script. In such cases, update the ``dracut`` configuration in the osimage to include the desired filesystem support and utilities and then pack it up.
```
Comment (kostty) [inline doc/compute_storage_schemes.rst]: should be the other way round, the partitioning script is run within the ramdisk

File: doc/ha_design.rst [hunk 2]
```diff
@@ -5,15 +5,15 @@ TrinityX HA design and implementation
 Introduction
 ------------
 
-The TrinityX installation playbooks can set up the controller, or the controller pair, either as a regular stand-alone system, or as part of a High-Availability pair with failover of services between the two controllers of the pair.
+The TrinityX installation playbooks can set up the controller, or the controller pair, either as a regular stand-alone system, or as part of a High-Availability pair with failover of services between the two controllers.
```
Comment (kostty) [inline doc/ha_design.rst]: The original wording is somewhat vague, can be interpreted like there are 4 options of how to configure controller nodes, whereas in fact there are just two - non-HA and HA.

File: doc/hintsntips_luna.rst [hunk 3]
```diff
@@ -37,16 +37,16 @@ The easiest way to get a new image for modifications is to clone an existing one
...
 ~~~~~~~~~~~~~~~~~~~~
 
-In some other cases you may prefer to start from a fresh image. In that case copy the existing playbook and change name of the image (``image_name`` variable and ``hosts`` target)::
+In some cases it may be preferable to start from a fresh image. In that case, copy the existing playbook and change the name of the image (``image_name`` variable and ``hosts`` target)::
 
     # cp compute.yml new-compute.yml
     # sed -i -e 's/compute/new-compute/' new-compute.yml
 
-You might also want to customize your new image to add more roles. When done, run ``ansible-playbook``::
+In the event an image has been customized/more roles added, run ``ansible-playbook`` once finished::
```
Comment (Marc69) [inline doc/hintsntips_luna.rst]: Suggestion:
'This new yml file can be customized by adding more roles.

- Now run ansible-playbook::'
(new line for a new important step in the procedure)

File: doc/hintsntips_luna.rst [hunk 4]
```diff
@@ -58,7 +58,7 @@ There are two approaches to creating new images in TrinityX: manually and via An
...
 
-Regarding the manual modification of an image's content: Luna has a tool called ``lchroot``. It is a wrapper around the good old ``chroot`` tool, but with some additions and integration with Luna. In particular, it mounts /dev, /proc, /sys filesystems on start and unmounts on exit. Please don't try to pack the image if you have an ``lchroot`` session running. Packing the image at this point will pack the content of the mentioned service filesystems and this is probably not what you want.
+Regarding the manual modification of an image's content: Luna has a tool called ``lchroot``. It is a wrapper around the good old ``chroot`` tool, but with some additions and integration with Luna. In particular, it mounts /dev, /proc, /sys filesystems on start and unmounts on exit. Don't try to pack the image if an ``lchroot`` session is running. Packing the image at this point will pack the content of the mentioned service filesystems and this is probably not desired.
 
 The tool can also mock the kernel version allowing to install software which requires particular release::
```
Comment (Marc69) [inline doc/hintsntips_luna.rst]: 'particular release': 'a particular release'

File: doc/hintsntips_luna.rst [hunk 5]
```diff
@@ -58,7 +58,7 @@ There are two approaches to creating new images in TrinityX: manually and via An
 
 In the manual approach, first clone the image with ``luna osimage clone`` and modify as a regular set of files or in chrooted environment.
```
Comment (Marc69) [inline doc/hintsntips_luna.rst]: 'modify it as a regular set of files or in a chroot environment'

File: doc/hintsntips_luna.rst [hunk 6]
```diff
@@ -276,9 +276,9 @@ By default, every group is created with the default partscript where the osimage
 	mkdir /sysroot/boot
 	mount /dev/sda1 /sysroot/boot
 
-Please note that it is not necessary to change the osimage in order to make a node diskful. You can use the same image, but instead of mounting the ramdisk to ``/sysroot`` we put /dev/sda2 there.
+Please note that it is not necessary to change the osimage in order to make a node diskful. The same image can be used, but instead of mounting the ramdisk to ``/sysroot``, /dev/sda2 is placed there.
 
-To make a node self-contained, we still need to add bootloader and change fstab to tell systemd where to find ``/``::
+To make a node self-contained, bootloader should be added and fstab changed to communiate to systemd where to find ``/``::
```
Comment (Marc69) [inline doc/hintsntips_luna.rst]: 'a bootloader should be added and fstab changed to communicate to systemd...'

File: doc/hintsntips_luna.rst [hunk 7]
```diff
@@ -146,14 +146,14 @@ Name is being used as domain for DNS. All IP addresses  which will be defined la
...
+A key concept in Luna is that of groups. Most (after osimage) of the customizations in Luna are performed here. A group is a homogeneous set of nodes. They usually have the same role within the cluster, with a similar hardware configuration, software set, and are connected to the same networks. Usually, they are logically grouped to the same queue (or partition) in the scheduling system. It is possible to specify the same osimage for several groups and perform additional customizations on install.
 
-Creatin group requires osimage to be specified. Group can't exist without image. Second mandatory parameter is network node is connected to. It is assuming that node need to be installed via network, otherwise it is make no much sense to use network provisioning tool::
+Creating a group requires the osimage to be specified. A group can't exist without an image or connection to a network node. It is assumed that nodes need to be installed via the network, as we are using a network provisioning tool::
```
Comment (kostty) [inline doc/hintsntips_luna.rst]: ...without an image or connection to a network ~~node~~.

File: doc/hintsntips_luna.rst [hunk 8]
```diff
@@ -247,23 +247,23 @@ This will open editor for your convenience where you can type config with regula
...
 
-Please note that you don't need to specify ``NAME=`` and ``DEVICE=`` for interfaces; ``IPADDR=`` and ``PREFIX=`` will be added automatically on per-node basis.
+Please note that it is unnecessary to specify ``NAME=`` and ``DEVICE=`` for interfaces; ``IPADDR=`` and ``PREFIX=`` will be added automatically on a per-node basis.
 
 
 Scripts in groups
 ~~~~~~~~~~~~~~~~~
 
-Sometimes install procedure need to be altered to performs some tasks before or after tarball with osimage will be placed. Here is the place where customization scripts go into play. Each group has 3 of them: prescript, partscript and postscript.
+Sometimes the installation procedure needs to be altered to perform some tasks before or after creating the tarball where the osimage will be placed. Customization scripts come into play here. Each group has 3: prescript, partscript, and postscript.
```
Comment (kostty) [inline doc/hintsntips_luna.rst]: >before or after creating the tarball where the osimage will be placed
rather, before or after the osimage will be deployed.

File: doc/hintsntips_luna.rst [hunk 9]
```diff
@@ -276,9 +276,9 @@ By default every group is being created with default partscript where osimage wi
 	mkdir /sysroot/boot
 	mount /dev/sda1 /sysroot/boot
 
-Please note, that we do not need to change osimage anyhow to make node diskful. We are using the same image, but istead of mounting ramdisk to ``/sysroot`` we are putting /dev/sda2 there.
+Please note that it is not necessary to change the osimage in order to make a node diskful. The same image can be used, but instead of mounting the ramdisk to ``/sysroot``, /dev/sda2 is placed there.
 
-To make node self-contained we still need to add bootloader and change fstab to tell systemd where to find ``/``::
+To make a node self-contained, bootloader should be added and fstab changed to communiate to systemd where to find ``/``::
```
Comment (kostty) [inline doc/hintsntips_luna.rst]: for later: _self-contained_ needs to be elaborated

File: doc/hintsntips_luna.rst [hunk 10]
```diff
@@ -292,57 +292,57 @@ To make node self-contained we still need to add bootloader and change fstab to
...
+Further individual settings for node are ``--setupbmc`` and ``--service``. These are mostly relevant for debugging. The first allows disabling of attempts to configure BMC, as it is known this configuration might be flaky. ``--service`` tunable can be handy if an engineer needs to debug boot issues. Nodes in this mode will not try to run the install script, but will stay in the initrd stage, configure 2 consoles (Alt+F1, Alt+F2), and try to set up IP addresses and run ssh daemon. In addition, it can be used to inspect the hardware configuration of the node before setup and wiping of data on disks.
 
-Another debug feature is a flag ``luna node show --script`` it accepts two options ``boot`` and ``script``.
```
Comment (kostty) [inline doc/hintsntips_luna.rst]: ...two options ``boot`` and ~~``script``~~ ``install``.

File: doc/hintsntips_luna.rst [hunk 11]
```diff
@@ -292,57 +292,57 @@ To make node self-contained we still need to add bootloader and change fstab to
...
     # snmpwalk -On -c public -v 1 SWITCH_IP .1.3.6.1.2.1.17.7.1.2.2.1
 
 It should list something like::
 
     .1.3.6.1.2.1.17.7.1.2.2.1.2.1.24.102.218.96.27.201 = INTEGER: 210
 
-Last 6 numbers is some MAC address in decimal format. See ``man luna`` for more information how to decrypt it.
+The last 6 numbers is a MAC address in decimal format. See ``man luna`` for more information on how to decrypt it.
 
-When Luna is able to get MAC addresses from switches it can display them in ``luna cluster listmacs``.
```
Comment (kostty) [inline doc/hintsntips_luna.rst]: ...it ~~can~~ will display them in ``luna cluster listmacs``.
I'd use a more affirmative tone

**Comment by kostty** [comment]
Looks good to me

**Pull request merged**